train.py

```diff
@@ -4,13 +4,26 @@
 from absl import app, flags, logging
```

Comment:
Since we're not training the model let's give a better name. Something like `initialize_model_demo.py`.